src/analyze/analyze-security.c

```diff
@@ -2805,10 +2815,9 @@ int analyze_security(sd_bus *bus,
                         fflush(stdout);
                 }
 
-                r = table_print(overview_table, stdout);
+                r = table_print_with_pager(overview_table, json_format_flags, /* pager_flags= */ 0, /* show_header= */ true);
```

Comment:
please pass `arg_pager_flags` from the main program through here too, instead of passing them as 0 here. (and similar above)